File: Listas de exercícios em python/Lista 2 - Casa/Exe 3.py
nota = float(input("Insira nota do aluno: "))

ParteInteira = int(nota)
resto = nota - ParteInteira

if resto > 0.5:
    NotaArredondada = ParteInteira + 1
else:
    NotaArredondada = ParteInteira

print(f"A nota arredondada do aluno é: {NotaArredondada}.")

File: Listas de exercícios em C/Lista 2 - Casa/Exe 3.c
#include <stdio.h>

int main() {
    float nota;
    int ParteInteira;
    float resto;
    int NotaArredondada;

    printf("Insira nota do aluno: ");
    scanf("%f", &nota);

    ParteInteira = (int)nota;
    resto = nota - ParteInteira;

    if (resto > 0.5) {
        NotaArredondada = ParteInteira + 1;
    } else {
        NotaArredondada = ParteInteira;
    }

    printf("A nota arredondada do aluno é: %d.\n", NotaArredondada);

    return 0;
}